app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -26,6 +26,11 @@ def textual_group_conditions
     h
   end
 
+  def textual_group_smart_management
+    items = %w(tags)
+    items.collect { |m| send("textual_#{m}") }.flatten.compact
+  end
```

Comment:
This method is exactly the same for every entity except the provider. 
Should it be somehow refactored into ```container_summary_helper.rb```?
@matthewd